File: python/copass-management/src/copass_management/tools/wire_integration_to_agent.py
from __future__ import annotations

from typing import Any, Dict, TYPE_CHECKING

if TYPE_CHECKING:
    from copass_management.registrar import ToolContext


async def wire_integration_to_agent(
    ctx: "ToolContext", input: Dict[str, Any],
) -> Any:
    agent_slug = str(input["agent_slug"])
    app_slug = str(input["app_slug"])
    return await ctx.client.agents.wire_integration(
        ctx.sandbox_id, agent_slug, app_slug,
    )

File: typescript/packages/management/src/tools/wire_integration_to_agent.ts
import type { ToolContext, ToolHandler } from '../registrar.js';

export const wireIntegrationToAgent: ToolHandler = async (
  ctx: ToolContext,
  input: Record<string, unknown>,
) => {
  const agentSlug = String(input.agent_slug);
  const appSlug = String(input.app_slug);
  return ctx.client.agents.wireIntegration(ctx.sandboxId, agentSlug, appSlug);
};
